app/src/main/AndroidManifest.xml

```diff
@@ -12,8 +11,8 @@
     android:icon="@drawable/plaid_icon"
     android:label="@string/app_name"
     android:supportsRtl="true"
-    android:theme="@style/AppTheme">
-
+    android:theme="@style/AppTheme"
+    android:usesCleartextTraffic="true">
```

Comment:
we should have a comment here why we are using this and that it's probably not the best idea for them to use in their actual app

Reply:
Added comment